src/Components/Navbar/Navbar.jsx

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import MainLogo from "../../Icons/MainLogo";
+import CardIcon from "../../Icons/CardIcon";
+const Navbar = () => {
```

Comment:
this code shouldn't be here, please remove it

Reply:
should I remove the Navbar component ? I didn't use it